server/vulnerabilities/nvd/cve_test.go

```diff
@@ -286,6 +313,9 @@ func TestTranslateCPEToCVE(t *testing.T) {
 		ds.DeleteOutOfDateVulnerabilitiesFunc = func(ctx context.Context, source fleet.VulnerabilitySource, duration time.Duration) error {
 			return nil
 		}
+		ds.DeleteOutOfDateVulnerabilitiesFunc = func(ctx context.Context, source fleet.VulnerabilitySource, duration time.Duration) error {
+			return nil
+		}
```

Comment:
This is the same as the 3 lines above.

Reply:
Thanks for catching this! (probably also pushed by the other two PRs that I merged yesterday)
[Fix PR comments](https://github.com/fleetdm/fleet/pull/14791/commits/7a97762a299cd7d537ac14f95d5c0db733c6c972)